authorjapps/zerocode
# ISS-244 - Dependency clean up for JUnit Jupiter Zerocode lib

### What to check in the PR?
+ Please check if anything else can be excluded
   + Please compare with the core `<artifactId>zerocode-tdd</artifactId>`
+ Please check any redundant exclusions made
    + i.e. something need not be excluded, but excluded

### How to validate?
+ Whether, we can add more exclusions to the list and see if the project is compiling as well as running fine(not breaking anything)
   + Issue this command `mvn clean install` by changing dir to `zerocode/junit5-testing`
+ Then we can take help of `mvn dependency:analyze-report` to check if anything is missing.
   + Sometimes this report is misleading, but gives a good suggession.
